### PR TITLE
Feature/dtfs2 4026 automatic cover

### PR DESCRIPTION
**Pull request by eddieleeper**

### CODE REVIEW

File: gef-ui/server/controllers/automatic-cover/index.js [hunk 1]
```diff
@@ -39,10 +41,26 @@ const validateAutomaticCover = async (req, res, next) => {
     return receivedFields.some((field) => field === 'false');
   }
 
-  const { body, params } = req;
+  const { body, params, query } = req;
```
Comment (davehlewis) [inline gef-ui/server/controllers/automatic-cover/index.js:44]: Don't see query used anywhere?

**Review by davehlewis — APPROVED** [review]
Looks good to me, just a couple of questions

File: gef-ui/server/controllers/automatic-cover/index.js (hunk 1 above)
Comment (eddieleeper) [reply gef-ui/server/controllers/automatic-cover/index.js:44]: @davehlewis   It's used on line #46

Comment (davehlewis) [reply gef-ui/server/controllers/automatic-cover/index.js:44]: My bad - sorry not sure how I missed that!

File: e2e-tests/gef/cypress/integration/automatic-cover.spec.js [hunk 2]
```diff
@@ -56,9 +56,9 @@ context('Automatic Cover Page', () => {
     it('takes user to `not eligible for automatic cover` page if at least 1 FALSE field has been selected', () => {
       automaticCover.automaticCoverTerm().each(($el, index) => {
         if (index === 0) {
-          $el.find('[data-cy="automatic-cover-false"]').click();
+          $el.find('[data-cy="automatic-cover-false"]').trigger('click');
```
[the remark this answers is not in the document]
Comment (eddieleeper) [reply e2e-tests/gef/cypress/integration/automatic-cover.spec.js:59]: Good point - the whole file is like that - I decided to just change the deprecated call - without taking a step back.
Probably one for the refactoring list.

**Pull request merged**